src/server/server.js

```diff
@@ -40,10 +40,16 @@ class Server {
 			config.logging = Object.assign({}, config.logging, { level: 'error' });
 		}
 
-		// Validate profiles
-		let profileKeys = Object.keys(config.profiles);
-		let hasValidProfileConfiguration = profileKeys.every(profileKey => {
-			return loadProfile(profileKey, config.profiles[profileKey]);
+		// Validate profiles for any spec provided
+		let dstu2ProfileKeys = config.profiles.dstu2 ? Object.keys(config.profiles.dstu2) : [];
+		let stu3ProfileKeys = config.profiles.stu3 ? Object.keys(config.profiles.stu3) : [];
+
+		let hasValidDSTU2ProfileConfiguration = dstu2ProfileKeys.every(profileKey => {
+			return loadProfile(profileKey, config.profiles.dstu2[profileKey]);
+		});
+
+		let hasValidSTU3ProfileConfiguration = stu3ProfileKeys.every(profileKey => {
+			return loadProfile(profileKey, config.profiles.stu3[profileKey]);
```

Comment:
We can use the same `loadProfile()` function for either spec?  I guess since we haven't actually implemented STU3 yet we can always change it later if we need to.

Reply:
Yes, it actually just loads a file and stores it for later use under `config.profiles[spec]serviceModule`. The name could be updated since it's purpose is to verify that the service config provided exists and can be loaded.  If there was time, there is a TODO in there to check the loaded service against an interface and make sure the required functions are present.